python/ray/scripts/scripts.py

```diff
@@ -622,6 +622,13 @@ def debug(address: str, verbose: bool):
     type=str,
     help="a JSON serialized dictionary mapping label name to label value.",
 )
+@click.option(
+    "--include-log-monitor",
+    default=None,
+    type=bool,
+    help="If set to True or left unset, a log monitor will start to monitor "
+    "the log files of all processes on this node and push their contents to GCS.",
```

Comment:
Nit: Wondering should we clarify that to avoid duplication, only one of the worker or head node should have this value to be true on the same physical host?

Reply:
added!